examples/onchain-signer/contracts/Gasless.sol

```diff
@@ -9,22 +9,15 @@ struct EthereumKeypair {
     uint64 nonce;
 }
 
-struct EthTx {
-    uint64 nonce;
-    uint256 gasPrice;
-    uint64 gasLimit;
-    address to;
-    uint256 value;
-    bytes data;
-    uint256 chainId;
-}
-
 // Proxy for gasless transaction.
 contract Gasless {
     EthereumKeypair private kp;
 
-    function setKeypair(EthereumKeypair memory keypair) external payable {
+    constructor (EthereumKeypair memory keypair) payable {
```

Comment:
Contract create transactions are usually unencrypted for verification, so that's why I decided to have the `setKeypair` setter.